minkindr_conversions/test/kindr_msg_test.cc

```diff
@@ -0,0 +1,70 @@
+#include "minkindr_conversions/kindr_msg.h"
+
+#include <glog/logging.h>
+#include <gtest/gtest.h>
+
+#include "testing_predicates.h"
+
+namespace tf {
+
+TEST(KindrMsgTest, poseKindrToMsgToKindr) {
+  Eigen::Quaterniond rotation(Eigen::Vector4d::Random());
+  rotation.normalize();
+  Eigen::Vector3d position(Eigen::Vector3d::Random());
+  kindr::minimal::QuatTransformation kindr_transform(rotation, position);
+
+  geometry_msgs::Pose msg;
+  poseKindrToMsg(kindr_transform, &msg);
+  kindr::minimal::QuatTransformation output_transform;
+  poseMsgToKindr(msg, &output_transform);
+
+  EXPECT_NEAR_EIGEN(output_transform.getRotation().toImplementation().coeffs(),
+                    rotation.coeffs(), 1e-6);
```

Comment:
since we don't do math, this tolerance is very generous. Actually, shouldn't they be binary-equal since it's just copying of the coefficients? (as long as no extra normalization or the like is going on...)
would be good to make it a named constant.